lib/MusicBrainz/Server/Email.pm

```diff
@@ -524,20 +524,20 @@ $message
 EOF
     }
 
-    my $admin_addresses = join ', ', map { _user_address($_) } @{ $opts{admins} };
```

Comment:
`$opts{admins}` is no longer used, so you can remove the two lines from https://github.com/metabrainz/musicbrainz-server/blob/3caba434099a225efcac614bda92ecf6709a1a20/lib/MusicBrainz/Server/Controller/User.pm#L524-L530 that fetch them and pass them in.

Reply:
Should be fixed as of https://github.com/metabrainz/musicbrainz-server/commit/a2931c07aaf105db13e5d0c290e32873a4f669e8